packages/dashboard-core-plugins/src/panels/CommandHistoryPanel.tsx

```diff
@@ -105,6 +106,11 @@ class CommandHistoryPanel extends Component<
     }
   }
 
+  handleShow(): void {
+    // virtual list requires a force re-render when picked up and dropped in same place
+    this.container.current?.update();
```

Comment:
Does show always get called in this case? What about listening to `onResize` instead?

Reply:
show is enough